dependencies: Remove and add dependencies

gorozco1 opened the pull request:
* cc-runtime: Required qemu-lite which is not packaged for RHEL and
  CentOS. This should not be a requeriment for those distributions.

  * cc-proxy: Required clear-containers-selinux which should be an
  optional package for systems with selinux enabled and only by sysadmin
  desition.

  *  cc-runtime did not install cc-proxy and cc-shim when is installed on
  debian based distros.

  Fixes: #6
  Fixes: #7
  Fixes: #10 

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

gorozco1 commented:
@chavafg new packages are in http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3-staging/

chavafg commented:
@gorozco1, tried new packages and #7 is solved, but I still get qemu-lite dependency error. 

```
[cloud@centos-obs-cc3 ~]$ sudo yum install cc-runtime
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: bay.uchicago.edu
 * extras: centos.mirror.ndchost.com
 * updates: centos.sonn.com
Resolving Dependencies
--> Running transaction check
---> Package cc-runtime.x86_64 0:3.0.0alpha.1+git.734bdd9-25.1 will be installed
--> Processing Dependency: clear-containers-image for package: cc-runtime-3.0.0alpha.1+git.734bdd9-25.1.x86_64
--> Processing Dependency: qemu-lite for package: cc-runtime-3.0.0alpha.1+git.734bdd9-25.1.x86_64
--> Running transaction check
---> Package cc-runtime.x86_64 0:3.0.0alpha.1+git.734bdd9-25.1 will be installed
--> Processing Dependency: qemu-lite for package: cc-runtime-3.0.0alpha.1+git.734bdd9-25.1.x86_64
---> Package clear-containers-image.x86_64 0:16020-28.1 will be installed
--> Finished Dependency Resolution
Error: Package: cc-runtime-3.0.0alpha.1+git.734bdd9-25.1.x86_64 (home_clearcontainers_clear-containers-3-staging)
           Requires: qemu-lite
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

gorozco1 committed:
dependencies: cc-oci-runtime remove qemu-lite dep in CentOS

cc-runtime: Required qemu-lite which is not packaged for RHEL and
CentOS. This should not be a requeriment for those distributions.

Fixes: #6

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

gorozco1 committed:
dependencies:  cc-proxy requirement selinux rules

cc-proxy: Required clear-containers-selinux which should be an
optional package for systems with selinux enabled and only by sysadmin
desition.

Fixes: #7

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

gorozco1 committed:
dependencies: Add cc-proxy and cc-shim as dependencies

cc-runtime did not install cc-proxy and cc-shim when is installed on
debian based distros.

Fixes: #10

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

chavafg commented:
@gorozco1, @erick0z, with latest changes, I verified that issues #6, #7 and #10 are fixed.

chavafg commented:
lgtm
